src/index.html

```diff
@@ -8,6 +8,71 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class = "stars stars--0">
+      <div class = "stars__star"></div>
+
+      <div class = "stars__star"></div>
+
+      <div class = "stars__star"></div>
+
+      <div class = "stars__star"></div>
+
+      <div class = "stars__star"></div>
```

Comment:
```suggestion
      <div class = "stars__star"></div>
      <div class = "stars__star"></div>
      <div class = "stars__star"></div>
      <div class = "stars__star"></div>
      <div class = "stars__star"></div>
```

Redundant empty line between child elements